README.md

```diff
@@ -19,3 +19,78 @@ These files are used as sample data in xwrf testing suites, tutorials, and docum
 - [`tiny.nc`](./data/tiny.nc): A tiny file for testing purposes
 - [`met_em.d01.2005-08-28_12:00:00.nc`](./data/met_em.d01.2005-08-28_12:00:00.nc): Metgrid ouput from [openwfm](https://wiki.openwfm.org/wiki/How_to_run_WRF-Fire_with_real_data)
 - [`wrfout_d03_2018-10-16_00:00:00.nc`](./data/wrfout_d03_2018-10-16_00:00:00.nc): Custom WRF output file for testing purposes
+
+## Sample Intake Catalog
+
+We also include an [intake](https://github.com/intake/intake) catalog, which makes use of [intake-xarray](https://github.com/intake/intake-xarray).
+
+You can access this data catalog using the following syntax:
+
+```python
+import intake
+
+# Read in the data catalog from github
+cat = intake.open_catalog('https://raw.githubusercontent.com/mgrover1/sample-wrf-data/main/catalog.yml')
+
+# Read one of the sample datasets
+ds = cat["xwrf-sample-ssp245"].to_dask()
+
+ds
```

Comment:
I'd suggest to split the python code and output into separate blocks

Reply:
Ahh - good point! Can do!